contrib/devtools/README.md

```diff
@@ -40,6 +40,12 @@ would be changed to:
 
 ```// Copyright (c) 2009-2015 The Bitcoin Core developers```
 
+gen-manpages.sh
+===============
+
+A small script to automatically create manpages in ../../doc/man by running the release binaries with the -help option.
+This requires help2man which can be found at: https://www.gnu.org/software/help2man/
```

Comment:
@theuni 

Oh ok now I get it, I was playing around with something like this:

```
bitcoind.1: $(top_builddir)/src/bitcoind.cpp
        -$(HELP2MAN) --no-info --version-string=$(PACKAGE_VERSION) \
          --output=$@ --name='manual page for $(BITCOIN_DAEMON_NAME) $(PACKAGE_VERSION)' \
          $(top_builddir)/src/$(BITCOIN_DAEMON_NAME)$(EXEEXT)
        -sed -i 's/\-$(shell git rev-parse HEAD | cut -c1-7)//g' $@

bitcoin-cli.1: $(top_builddir)/src/bitcoin-cli.cpp
        -$(HELP2MAN) --no-info --version-string=$(PACKAGE_VERSION) \
          --output=$@ --name='manual page for $(BITCOIN_CLI_NAME) $(PACKAGE_VERSION)' \
          $(top_builddir)/src/$(BITCOIN_CLI_NAME)$(EXEEXT)
        -sed -i 's/\-$(shell git rev-parse HEAD | cut -c1-7)//g' $@

bitcoin-tx.1: $(top_builddir)/src/bitcoin-tx.cpp
        -$(HELP2MAN) --no-info --version-string=$(PACKAGE_VERSION) \
          --output=$@ --name='manual page for $(BITCOIN_TX_NAME) $(PACKAGE_VERSION)' \
          $(top_builddir)/src/$(BITCOIN_TX_NAME)$(EXEEXT)
        -sed -i 's/\-$(shell git rev-parse HEAD | cut -c1-7)//g' $@

dist_man1_MANS=bitcoind.1 bitcoin-cli.1 bitcoin-tx.1

if ENABLE_QT
bitcoin-qt.1: $(top_builddir)/src/qt/bitcoin.cpp
        -$(HELP2MAN) --no-info --version-string=$(PACKAGE_VERSION) \
          --output=$@ --name='manual page for $(BITCOIN_GUI_NAME) $(PACKAGE_VERSION)' \
          $(top_builddir)/src/qt/$(BITCOIN_GUI_NAME)$(EXEEXT)
        -sed -i 's/\-$(shell git rev-parse HEAD | cut -c1-7)//g' $@
dist_man1_MANS+=bitcoin-qt.1
endif
```

Because my hope was to automate it completly, so that maybe the gitian build process generates the manpages in the end.It works pretty ok, but a huge problem is that running help2man on bitcoin-qt needs access to a running X-Server -_-.

So someone needs to generate and commit the manpages anyway. May as well than use something like your maketarget, at least this is much more elegant than my shell script... :)

Going to look into it, thanks!